packages/near-fast-auth-signer/src/utils/multi-chain/chains/Bitcoin.ts

```diff
@@ -1,10 +1,10 @@
 import axios from 'axios';
```

Comment:
I believe there is a type issue


https://github.com/near/fast-auth-signer/assets/22358004/a26b508e-89a0-49b1-8ad3-231dc6bbb417

I think when you call `attatchGasAndNonce` you should parse the value to ethers `parseEthers(data.value)`